docs/modules/help/pages/reporting_services.adoc

```diff
@@ -0,0 +1,13 @@
+= Accessing the Services Report
+
+WARNING: Depending on your access rights, your actual screen might differ from what is shown in the screenshots in this help system.
+
+[[overview]]
+== Overview
+
+This screen gives you an overview of all the services currently managed on your behalf.
```

Comment:
This screen is more a billing view, it shows all services including their price and also special deals. It's the actual source which gets used to create invoices. What's here will be on the invoice.